### PR TITLE
Added max_min sum in array 

### PR DESCRIPTION
**Pull request by aditisneh**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes : )

### Description

Added max min sum in array in java. This calculates the minimum and maximum sum of 4 elements of the array comprising of 5 elements.

### Checklist

- [y] I've read the contribution guidelines.
- [y ] I've checked the issue list before deciding what to submit.
- [ y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#5278

### CODE REVIEW

File: Java/other/max_min_sum.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/*Given five positive integers, find the minimum and maximum values 
+that can be calculated by summing exactly four of the five integers. 
+Then print the respective minimum and maximum values as a single line of two space-separated long integers.*/
+import java.io.*;
+
+import java.util.*;//impoting java packages
+
+public class sol{
```
Comment (gat786) [inline Java/other/max_min_sum.java]: Class name can be improved

File: Java/other/max_min_sum.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  
+    public static void main(String[] args){
+      
+        Scanner in = new Scanner(System.in);
+      
+        long arr[] = new long[5];
```
Comment (gat786) [inline Java/other/max_min_sum.java]: You have mentioned integers in the description but have used long here.

**Review by gat786 — CHANGES_REQUESTED** [review]

File: Java/other/max_min_sum.java (hunk 2 above)
Comment (aditisneh) [reply Java/other/max_min_sum.java]: Okay, will change the description then, long is used to make the code more acceptable to variety of cases.

File: Java/other/max_min_sum.java (hunk 1 above)
Comment (aditisneh) [reply Java/other/max_min_sum.java]: Okay, changing it to max_min_sum. Thanks for reviewing :)

File: Java/other/max_min_sum.java (hunk 2 above)
Comment (aditisneh) [reply Java/other/max_min_sum.java]: Hey, I have done the changes, do view it and let me know for further improvement.

Comment (gat786) [reply Java/other/max_min_sum.java]: Okay.

File: Java/other/max_min_sum.java (hunk 1 above)
Comment (gat786) [reply Java/other/max_min_sum.java]: Class names in Java are mostly PascalCased.

Comment (aditisneh) [reply Java/other/max_min_sum.java]: Okay, doing that.

File: Java/README.md [hunk 3]
```diff
@@ -333,6 +333,7 @@ _add list here_
 - [Tower of Hanoi problem](other/TowerOfHanoi.java)
 - [Unique 3](other/UniqueIII.java)
 - [LastdigitofnthFibonacci](other/LastDigitFibonacci.java)
+- [MaxMinSum](other/max_min_sum.java)
```
Comment (ankitaggarwal23) [inline Java/README.md]: make it readable and understandable

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top
- Use proper code Format 
- Remove extra lines of code and spaces
- Add sample I/O at the end of the Code
- Add Time and Space Complexity at the end of the Code   
- Add new line at the EOF

Use proper display message how you are reading and writing outputs

Add input-output prompts to make the program more interactive.

Shift the logic to another function and use main for driver code

You need to add comments to your code. Make sure the comments you add are above the line of code and not alongside it.

File: Java/README.md (hunk 3 above)
Comment (aditisneh) [reply Java/README.md]: I have done all the requested changes, kindly look into that, thank you.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.

**Comment by aditisneh** [comment]
> Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.

In which channel, I am not getting.

**Comment by aditisneh** [comment]
> Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.

In which channel?

**Comment by ankitaggarwal23** [comment]
> > Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.
> 
> In which channel?

Gssoc server #neo-algo

**Comment by ankitaggarwal23** [comment]
> > > Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.
> > 
> > 
> > In which channel?
> 
> Gssoc server #neo-algo

Are you a gssoc participant or not?

**Comment by aditisneh** [comment]
> > > > Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.
> > > 
> > > 
> > > In which channel?
> > 
> > 
> > Gssoc server #neo-algo
> 
> Are you a gssoc participant or not?

I am a gssoc participant, but in the discord server, i am not able to see any project channel, only announcements and faq.

**Comment by ankitaggarwal23** [comment]
> > > > > Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.
> > > > 
> > > > 
> > > > In which channel?
> > > 
> > > 
> > > Gssoc server #neo-algo
> > 
> > 
> > Are you a gssoc participant or not?
> 
> I am a gssoc participant, but in the discord server, i am not able to see any project channel, only announcements and faq.

It means you didn't read any followed gssoc instruction just read it properly and it will add all the channel

**Comment by ankitaggarwal23** [comment]
Closing this due to inactivity. Thank you for your valuable time.😊

**Review by ankitaggarwal23 — COMMENTED** [review]
Closing this due to inactivity. Thank you for your valuable time.😊

**Pull request closed without merging**